android/lib/src/main/java/com/swmansion/gesturehandler/Extensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.swmansion.gesturehandler
+
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.modules.core.DeviceEventManagerModule
+import com.facebook.react.uimanager.UIManagerModule
+
+val ReactContext.deviceEventEmitter: DeviceEventManagerModule.RCTDeviceEventEmitter
+    get() = this.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
+
+val ReactContext.UIManager: UIManagerModule
+    get() = this.getNativeModule(UIManagerModule::class.java)!!
```

Comment:
Missing newline at the end of the file.

Reply:
Fixed in 6635db93e7cf7baa7f9ca3e8eeed35bf41cd9aee